deno/bundle.ts

```diff
@@ -3,4 +3,14 @@ import { writeStage2 } from './lib/stage2.ts'
 const [payload] = Deno.args
 const { basePath, destPath, externals, functions, importMapData } = JSON.parse(payload)
 
-await writeStage2({ basePath, destPath, externals, functions, importMapData })
+try {
+  await writeStage2({ basePath, destPath, externals, functions, importMapData })
+} catch (error) {
+  if (error instanceof Error) {
+    if (error.message.includes("The module's source code could not be parsed")) {
```

Comment:
Could we not have a single `if` here?

```js
if (error instanceof Error && error.message.includes("The module's source code could not be parsed")) {
  // (...)
}
```

Reply:
done in 10bbe187cd6b8b727ce0d9fedc2ecf2996350ad2